servicemigration/port.py

```diff
@@ -0,0 +1,53 @@
+import copy
+
+default = {
+    "PortAddr": "",
+    "Protocol": "",
+    "Enabled": False,
+    "UseTLS": False
+}
+
+def deserialize(data):
+    """
+    Deserializes the list of Ports
+    :param data:
+    :return: a list of Port instances
+    """
+    ports = []
+    for p in data:
```

Comment:
We should protect against data = None here:
    if data is None:
        return []